types/tpdirect/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
```

Comment:
I was not sure if that would work, for example 'jquery` lists only `tests` files there, not ones added via triple slash notation.

Reply:
I use `/// < reference path` because I saw @types/jquery used `/// < reference path`.  
Because this definition is similar to jquery, so I reference the code of @types/jquery. 
But, I've tried it works with files of tsconfig.json.

Should I use `/// < reference path` instead of `files`?
I'm not sure which pattern is better.

Reply:
I find it a little easier to read when they're all in one place but, as long as you only reference files in this types package, they're more or less equivalent.  I'm fine with either.